packages/pigeon/lib/generator_tools.dart

```diff
@@ -351,13 +352,23 @@ Map<TypeDeclaration, List<int>> getReferencedTypes(
   return references.map;
 }
 
+bool _canContainAnyType(TypeDeclaration type) {
```

Comment:
Could you add a comment explaining what this means exactly? Contains seems ambiguous here, since for `Object` that means the variable can contain anything, but for the collections I assume fro the implementation that means the *collection* can contain anything, which isn't actually the same thing. (e.g., `List foo` can't "contain any type" in the same sense that `Object foo` can, because `List foo = someMap` isn't valid).

Reply:
Renamed and added docstring.